src/store/modules/swap/ethBancor.ts

```diff
@@ -1469,6 +1469,11 @@ export class EthBancorModule
   get stats() {
     const apiData = this.apiData!;
     const bntUsdPrice = Number(apiData.bnt_price.usd);
+    const bntPrice24Change = Number(
+      ((bntUsdPrice / Number(apiData.bnt_price_24h_ago.usd) - 1) * 100).toFixed(
```

Comment:
Export this logic to a pureFunction and create a couple tests from Asafs [comment](https://github.com/bancorprotocol/webapp/issues/1050#issue-811127591)

Reply:
Yeah was planning to its just a draft, waiting on the API to give the rest of the data.